toolkit/components/uniffi-bindgen-gecko-js/src/templates/Scaffolding.cpp

```diff
@@ -66,7 +70,7 @@ Args PrepareArgs({{ func.input_arg_list() }}, mozilla::ErrorResult& aUniFFIError
 // For async calls this should be called in the worker thread
 Result Invoke(Args& aArgs) {
     Result result = {};
-    result.mReturnValue = ::{{ func.rust_name() }}(
+    {%- if func.has_return_type() %}result.mReturnValue = {%- endif %}::{{ func.rust_name() }}(
```

Comment:
ditto the incomplete type, had to remove the `mReturnValue` when there is no return value